src/routes/CallbackPage.tsx

```diff
@@ -1,11 +1,32 @@
 import { memo } from 'react';
 import { Callback } from '@deriv-com/auth-client';
 
+type TTokens = {
+    [key: string]: string | undefined;
+};
+
+const groupTokens = (tokens: TTokens) => {
+    const grouped: { acct?: string; cur?: string; token?: string }[] = [];
+
+    for (let i = 1; i <= 3; i++) {
```

Comment:
There can be more than 3 accounts actually

Reply:
ah alright, will amend this